packages/repack/package.json

```diff
@@ -60,6 +60,7 @@
   "dependencies": {
     "@callstack/repack-dev-server": "^1.0.1",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
+    "cli-progress": "^3.12.0",
```

Comment:
Please install `@types/cli-progress` in `devDependencies` because the library code is not compiling.